stix2/exceptions.py

```diff
@@ -49,3 +49,10 @@ class ImmutableError(STIXError, ValueError):
 
     def __init__(self):
         super(ImmutableError, self).__init__("Cannot modify properties after creation.")
+
+
+class VersioningError(STIXError, ValueError):
+    """Execption while using the Versioning API"""
+
+    def __init__(self, msg):
```

Comment:
Yes, there's a lot more instances of `field` than I realized. Oops. Let's do the right thing (use "Property" here) and I'll make a new issue to go back and clean up the other ones.